app/controllers/api/v1/admin/site_settings_controller.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Admin
+      class SiteSettingsController < ApiController
+        def index
```

Comment:
Do we need the `#index` here?

Reply:
Yeah I was told my Ahmad since retrieving the site settings is only used in the Admin Panel right now